Jenkinsfile

```diff
@@ -24,8 +24,8 @@ pipeline {
         ARTIFACTORY_CREDS       = 's_jenkins_epds'
         ARTIFACTORY_PYPI_REPO   = 'yr-python'
         PROJECT_NAME            = 'resdev'
-        BUILD_BRANCH            = '142_jenkinsFile_RAP'  // Any commits to this branch will create a build in artifactory
-        BUILD_TAG               = 'v*'  // Any commits tagged with this pattern will create a build in artifactory
+        BUILD_BRANCH            = 'main'  // Any commits to this branch will create a build in artifactory
+        BUILD_TAG               = 'release-*'  // Any commits tagged with this pattern will create a build in artifactory
```

Comment:
I think this is the only bit of the pull request we need.

Reply:
Correct. Hopefully updated accordingly